app/lib/frontend/templates.dart

```diff
@@ -567,26 +575,24 @@ abstract class PageLinks {
 
     final bool hasPrevious = currentPage > 1;
     results.add({
-      'state': {'state': hasPrevious ? null : 'disabled'},
```

Comment:
Why the churn here?

Reply:
I'm switching to simpler template preparation and mustache structure (esp. for the new pagination).